### ISSUE TITLE
Some localities are missing whosonfirst-data-locality-1542073707 extract

### ISSUE THREAD

**Issue by Joxit** [whosonfirst/go-whosonfirst-dist#9]
Hi,

I don't know if it is the right place for this, but in the latest `meta` extract from https://dist.whosonfirst.org/. there are some missing localities (like [Rennes](https://spelunker.whosonfirst.org/id/101751181/) or [Berlin](https://spelunker.whosonfirst.org/id/101748799/)) :confused:

**Comment by Joxit** [comment]
In fact I was using bundle, this one : `whosonfirst-data-locality-latest.tar.bz2` `f8d838bb6befc8ee8612081db0f0dd5308e93c083e05415a8c71843db7d7cf69`

I just noticed that `whosonfirst-data-locality-1542073707.tar.bz2` has data from the same date but is bigger...

**Comment by thisisaaronland** [comment]
Ah, okay. Yes, this is a side-efffect of the problems building the SQLite distributions from which the bundles are derived. I will rebuild the bundles tonight. 

(The SQLite distributions _should_ be all better as of yesterday (`e90e3003a7a8e2a11aab32372de47b82157a1e7bc7e7467dd5fb1c6d440aebed`))

Apologies for the inconvenience.

**Comment by straup** [comment]
FYI: https://github.com/whosonfirst/go-whosonfirst-dist/commit/2082a1fd03c69de66923fb1254b03fcfb31ea930

**Comment by Joxit** [comment]
It's ok, I rebuild meta files from git sources and `wof-dist-build -build-meta` command line.
Thank you for your quick response :smile: 
I'm using wof for [pelias](https://github.com/pelias/) and they use bundles for wof import and not sqlite

**Comment by thisisaaronland** [comment]
Yeah, there are still lots of reasons for the bundles so we'll make sure they work. The challenge is just that these tools aren't / can't be the focus of my "work" hours these days and the recent addition of all the Geonames localities to `whosonfirst-data` has meant that some of the assumptions (like running parallel instances of `bzip2`...) need to be revisited.
